frontend/test/model/EBEventTest.scala

```diff
@@ -79,10 +79,6 @@ class EBEventTest extends PlaySpecification {
       updateEventWithDescription(ebLiveEvent, "<!-- main-image: https://media.gutools.co.uk/images/e5b35b1f20588172b24960314cbf8e9c8482d3bf?crop=27_0_1800_1080 ! -->")
         .mainImageUrl must beSome(Uri.parse("https://media.gutools.co.uk/images/e5b35b1f20588172b24960314cbf8e9c8482d3bf?crop=27_0_1800_1080"))
     }
-    "should not blow up even if the url is really bizarre" in {
-      updateEventWithDescription(ebLiveEvent, "<!-- main-image: http://test.net/## -->")
-        .mainImageUrl must beNone
-    }
```

Comment:
Rather than removing the test, just remove the assertion, ie the `.mainImageUrl must beNone` part.

The history of this test is that the event-admin staff were occasionally accidentally entering malformed urls, which would crash the whole site. All we really care about is that parsing weird urls doesn't cause an exception  - we don't care whether it evaluates as `None` or not.